src/docs/src/conf.py

```diff
@@ -29,9 +30,13 @@
 
 nitpicky = True
 
-# should be over-written using rebar-inherited settings
-version = "3.3"
-release = "3.3.0"
+# load version numbers from version.mk
+version_file = Path().absolute().joinpath('../../../version.mk').resolve()
+with open(version_file) as file:
+    props = dict(line.strip().split('=', 1) for line in file)
```

Comment:
`black` also complains about the single quotes around the equal sign. Try changing to `"="`.